extensions/amp-a4a/rtc-documentation.md

```diff
@@ -84,9 +85,10 @@ The value of `rtc-config` must conform to the following specification:
         *   Vendors can use the same macros as other vendors.
 *   `urls`
     *   Optional parameter
-    *   Type: Array
-    *   Each value in the array must be a valid RTC endpoint URL. These are the custom URLs mentioned above.
+    *   Type: Array of strings or objects.
+    *   Each value in the array must be a valid RTC endpoint URL, or an object that contains a `url` and an `errorReportingUrl`. The array can be a mix of both of these types, as seen in the example above. In the case that an object is specified, the `url` within this object is treated equivalently as if it had been specified directly within the array, and errors from callouts to that URL are sent to its corresponding **errorReportingUrl**. The URLs specified here are the "custom URLs" mentioned above and throughout this document.
```

Comment:
contains a `url` and an `errorReportingUrl` as url strings.  Note that the urls must be secure (e.g. start with https://).

Reply:
done